tests/passes/simplification/control_flow_raising_test.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2019-2024 ETH Zurich and the DaCe authors. All rights reserved.
+
+import dace
+import numpy as np
+from dace.sdfg.state import ConditionalBlock
+from dace.transformation.pass_pipeline import FixedPointPipeline, Pipeline
+from dace.transformation.passes.simplification.control_flow_raising import ControlFlowRaising
+
+
+def test_dataflow_if_check():
+
+    @dace.program
+    def dataflow_if_check(A: dace.int32[10], i: dace.int64):
+        if A[i] < 10:
+            return 0
+        elif A[i] == 10:
+            return 10
+        return 100
+
+    sdfg = dataflow_if_check.to_sdfg()
+
+    assert not any(isinstance(x, ConditionalBlock) for x in sdfg.nodes())
+
+    ppl = FixedPointPipeline([ControlFlowRaising()])
+    ppl.__experimental_cfg_block_compatible__ = True
```

Comment:
ugh. Still necessary?

Reply:
Only until #1676 is ready and merged, then not anymore.. :-)